src/plots/plots.js

```diff
@@ -2828,6 +2828,15 @@ plots.doCalcdata = function(gd, traces) {
         );
     }
 
+    // clear relinked cmin/cmax values in shared axes to start aggregation from scratch
+    for(var k in fullLayout._colorAxes) {
+        var cOpts = fullLayout[k];
+        if(cOpts.cauto !== false) {
+            delete cOpts.cmin;
+            delete cOpts.cmax;
```

Comment:
Should we do something similar for `cmid`?

Reply:
No, we don't need that.

`cmid` is not relinked during the [cross-trace-defaults](https://github.com/plotly/plotly.js/blob/513491e6a8f20e247881609eefa74ffdb6c7c319/src/components/colorscale/cross_trace_defaults.js#L23-L47) and we don't aggregate its value during `calc`, like we do for `cmin` and `cmax` :arrow_heading_down: 

https://github.com/plotly/plotly.js/blob/513491e6a8f20e247881609eefa74ffdb6c7c319/src/components/colorscale/calc.js#L37-L38

https://github.com/plotly/plotly.js/blob/513491e6a8f20e247881609eefa74ffdb6c7c319/src/components/colorscale/calc.js#L47-L48